newIDE/app/src/SceneEditor/index.js

```diff
@@ -559,6 +571,19 @@ export default class SceneEditor extends React.Component<Props, State> {
     this.updateToolbar();
   };
 
+  _addNewObjectAndInstance = (newObjectName: string) => {
+    if (!this.state.addFirstInstancePosition.length) {
```

Comment:
Use null and change this condition to `if (!this.state.addFirstInstancePosition)`